packages/contracts-core/contracts/test/utils/NomadCore.sol

```diff
@@ -358,6 +358,10 @@ abstract contract NomadCore is Test {
     ) public {
         uint256 index = domainToIndex[domain];
         address newGovernorAddress;
+        // if domain is governorDomain, we don't need to enroll a remote router
```

Comment:
not the case. the governorDomain needs to enroll remote routers in order to send governance messages to remote routers

Reply:
plus, we should try to mirror our deploy code anyway, in which every router is enrolled in every other router